src/components/GalleryPage/GalleryMenu.js

```diff
@@ -16,9 +16,17 @@ function GalleryMenu({ hothNames, galleryLinks }) {
 	const handleClose = () => {
 		setAnchorEl(null);
 	};
+	const linkStyles = {
+		color: '#202020',
+		textDecoration: 'none'
+	};
+	const activeStyles = {
+		color: '#ff487e',
```

Comment:
Instead of using hardcoded hexcolor values, we actually have a defined color palette through MUI in `src/components/layout.js`. Try to use those as much as possible instead. For example, this would be `theme.palette.secondary.main`